addons/payment_stripe/models/payment_acquirer.py

```diff
@@ -27,6 +31,92 @@ class PaymentAcquirer(models.Model):
              "authenticate the messages sent from Stripe to Odoo.",
         groups='base.group_system')
 
+    # === ACTION METHODS === #
+
+    def action_stripe_connect_account(self, menu_id=None):
+        """ Create a Stripe Connect account and redirect the user to the next onboarding step.
+
+        If the acquirer is already enabled, close the current window. Otherwise, generate a Stripe
+        Connect onboarding link and redirect the user to it. If the link generation failed, redirect
+        the user to the acquirer form.
+
+        Note: This method is overridden by the internal module responsible for Stripe Connect.
+        Note: self.ensure_one()
+
+        :param int menu_id: The menu from which the user started the onboarding step, as an
+                            `ir.ui.menu` id.
+        :return: The next step action
+        :rtype: dict
+        """
+        self.ensure_one()
+
+        if self.state == 'enabled':
+            action = {'type': 'ir.actions.act_window_close'}
+        else:
+            # Account creation
+            connected_account = self._stripe_fetch_or_create_connected_account()
+
+            # Link generation
+            menu_id = menu_id or self.env.ref('payment.payment_acquirer_menu').id
+            account_link_url = self._stripe_create_account_link(connected_account['id'], menu_id)
+            if account_link_url:
+                action = {
+                    'type': 'ir.actions.act_url',
+                    'url': account_link_url,
+                    'target': 'self',
+                }
+            else:
+                action = self.env['ir.actions.actions']._for_xml_id(
+                    'payment.action_payment_acquirer'
+                )
```

Comment:
Is this really useful when we're apparently removing all the bits it provides?

The PR doesn't seem to edit `action_payment_acquirer`, and essentially all it's used for here is the model.